File: 1011 - volume_esfera/esfera.pas
program esfera(input,output);

uses 
    SysUtils;

function vesfera( var raio:Double ) : Double;
begin
    vesfera:= (4.0 / 3.0) * 3.14159 * raio * raio * raio;
end;

var
    raio, vol: Double;

begin
    readLn(raio);
    vol := vesfera(raio);
    writeln('VOLUME = ', Format('%.3f', [vol]));
end.

File: 1011 - volume_esfera/esfera.hs
import Text.Printf
import System.IO

vesfera :: Double -> Double
vesfera r = (4.0 / 3.0) * 3.14159 * r * r * r

main = do
    r <- readLn
    let vol = vesfera r :: Double
    printf "VOLUME = %.3f\n" vol